libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/LUIS/LuisAdaptiveRecognizer.cs

```diff
@@ -97,7 +98,11 @@ public override async Task<RecognizerResult> RecognizeAsync(DialogContext dialog
                 tempContext.TurnState[keyValue.Key] = keyValue.Value;
             }
 
-            return await wrapper.RecognizeAsync(tempContext, cancellationToken).ConfigureAwait(false);
+            var result = await wrapper.RecognizeAsync(tempContext, cancellationToken).ConfigureAwait(false);
+
+            this.TelemetryClient.TrackEvent("LuisResult", this.FillRecognizerResultTelemetryProperties(result, telemetryProperties, dialogContext.Context), telemetryMetrics);
```

Comment:
>   this.TelemetryClient [](start = 9, length = 23)

Why not dialogContext.TurnState.Get<TelemetryClient>() ?

Reply:
As per above, The TelemetryClient isn't currently added to TurnState. I saw your suggestion regarding having the middleware add it, but that telemetry middleware is currently optional and we want to ensure this telemetry lights up for folks who already have it enabled when they start to adopt adaptive.